Azure/communication-ui-library-ios
# [Feature] Implement the Local Avatar into Composite

## Purpose
- Include new Avatar picker into settings page
- Introduce AvatarManager
- Inject AvatarManager into the Setup View and Call View
- Loading avatar into the local participant 
<img width="502" alt="Screen Shot 2022-04-18 at 1 43 33 PM" src="https://user-images.githubusercontent.com/9044372/163875043-2d7692ee-e5a2-48a3-a2d5-524910d12a86.png">
<img width="502" alt="Screen Shot 2022-04-18 at 1 43 44 PM" src="https://user-images.githubusercontent.com/9044372/163875050-64e1d7b2-3ff8-4272-97c1-230a20b45d99.png">
<img width="502" alt="Screen Shot 2022-04-18 at 1 43 59 PM" src="https://user-images.githubusercontent.com/9044372/163875056-8ac21752-dad3-427a-aa9b-db1e1f8f38d1.png">
<img width="502" alt="Screen Shot 2022-04-18 at 1 44 11 PM" src="https://user-images.githubusercontent.com/9044372/163875058-c7ac883a-b7d2-4a94-87b1-793a69f5cd47.png">

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Step 1: Load up to sample app
Step 2: Go to settings page and pick an avatar
Step 3: Join a group call
Step 4: Notice Avatar in the set up view / call view
